packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+### Enhancements
+
+- Components: Make the `ProgressBar` public ([#61062](https://github.com/WordPress/gutenberg/pull/61062)).
```

Comment:
There's already an Enhancements section, we'll have to use it instead of adding a new one.

Reply:
Oh, thanks for pointing that out.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/61062/commits/47183b7d3fecf89869d23b43a49865e463417fa9.